src/main/java/de/ace/html2pdf/config/exception/PdfException.java

```diff
@@ -0,0 +1,31 @@
+package de.ace.html2pdf.config.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Slf4j
+@ResponseStatus(value = HttpStatus.I_AM_A_TEAPOT)
```

Comment:
This is redundant, as we pass the status code from the exception handler method.